src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -258,6 +251,65 @@ In the following, we briefly describe the different tuners proposed by the Multi
     - *MOCMAES search* is a multi-objective algorithm optimizing multiple tradeoffs simultaneously.
       To do that, MOCMAES employs a number of CMAES algorithms.
 
+===== Multi_Tuners_AutoML_V1
+
+*Workflow Overview:* generates at first the chosen values of hyperparameters for each trial. Then, it executes the different trials in parallel. Finally, it plots the different results using Visdom.
+
+*Specific variables:*
+
+.Multi_Tuners_AutoML_V1 specific variables
+[cols="2,5,2"]
+|===
+| *Variable name* | *Description* | *Type*
+| `VISDOM_ENABLED`
+|  If True, the results will be plotted in real time in the Visdom dashboard.
+| Boolean (default=True)
+|===
+
+image::MLOS-AutoML-V1.png[align=center]
+===== Multi_Tuners_AutoML_V2
+
+*Workflow Overview:* considers two execution pipelines (parallel and sequential).
+For all the tuning algorithms except the Bayesian one, the parallel pipeline is executed similar as `Multi_Tuners_AutoML_V1`. For the the bayesian one, the sequential one is executed. This pipeline both generates the hyperparameters' values and executes the associated trials sequentially.
```

Comment:
the the ==> the
Bayesian or bayesian. Please verify and use the same pattern in both situations.
You use a lot of `one`. For the bayesian one, ==> For the (B/b)ayesian algorithm, 
This pipeline both generates ==> This pipeline generates both

Reply:
I agree with your first comments.
For the last comment, both is for two verbs generates and executes.

Reply:
hmmm I see. In that case it would be nice to add a coma:
both generates the hyperparameters' values,  and executes the associated trials sequentially.